proxy/src/control_plane/errors.rs

```diff
@@ -8,7 +8,7 @@ use crate::error::{ErrorKind, ReportableError, UserFacingError};
 use crate::proxy::retry::CouldRetry;
 
 /// A go-to error message which doesn't leak any detail.
-pub(crate) const REQUEST_FAILED: &str = "Console request failed";
+pub(crate) const REQUEST_FAILED: &str = "Control plane request failed";
```

Comment:
Could you also update it in https://github.com/neondatabase/neon/blob/5d91d4e843009454ef92841c3f70232f46cf62bb/test_runner/random_ops/test_random_ops.py#L325
(we need to support both versions in that test for some time)

Reply:
Makes sense -- done in 696188a8ffda20f86e386a8ef27dded29a2d06c6